gci/students.py

```diff
@@ -160,9 +194,18 @@ def get_linked_students():
                     elif len(issue.assignees) > 1:
                         logger.info('task %d: Many assignees for %s: %s' %
                                     (task_id, url, ', '.join(issue.assignees)))
+
+                    if len(issue.assignees) != 1:
+                        user = get_closing_pr_author(url)
+                        if user is not None:
+                            student['username'] = user
+                            print('student %s is %s because of %s (from PR)' %
+                                  (student['id'], user, url))
+                            yield student
+                            break
                     else:
-                        student['username'] = issue.assignees[0]
+                        student['username'] = list(issue.assignees)[0]
```

Comment:
this seems to be unrelated to your issue.  I assume it is caused by the updated IGitt version, in which case you need a separate commit for the IGitt update and these fixes